File: Python/baekjoon/단계별로 풀어보기/정렬/10989_수 정렬하기 3.py
# https://www.acmicpc.net/problem/10989

from sys import stdin

input = stdin.readline
li = [0] * 10000
for _ in range(int(input())):
    li[int(input()) - 1] += 1
for i in range(10000):
    if li[i]:
        for _ in range(li[i]):
            print(i + 1)

File: C#/Baekjoon/정렬/10989_수 정렬하기 3.cs
//https://www.acmicpc.net/problem/10989

StreamReader sr = new StreamReader(new BufferedStream(Console.OpenStandardInput()));
StreamWriter sw = new StreamWriter(new BufferedStream(Console.OpenStandardOutput()));
int n = int.Parse(sr.ReadLine());
int[] a = new int[10001];
for (int i = 0; i < n; i++)
{
    a[int.Parse(sr.ReadLine())] += 1;
}
for (int i = 0; i < 10001; i++)
{
    for (int j = a[i]; j > 0; j--)
    {
        sw.WriteLine(i);
    }
}
sw.Close();
sr.Close();